src/app/Presentation.php

```diff
@@ -37,4 +38,18 @@ public function questions() {
     public function polls() {
         return $this->hasMany('App\Poll');
     }
+
+    /**
+     * List of users moderation the presentation (presenters included)
+     */
+    public function moderators() {
+        return $this->users()->where('role', 'presenter')->get();
+    }
+
```

Comment:
why adding this to your PR as it's already done in another issue, also you did it wrong as moderators have other roles than just presenter

Reply:
Added by mistake, used for testing without merging other branch 70bf08b